.github/workflows/onrelease-dotnet-build-amd64.yml

```diff
@@ -0,0 +1,22 @@
+name: Build amd64 Docker Image
```

Comment:
This workflow is now exactly the same as `build-arm64`, with the exception of the hardcoded architecture value. Could we collapse these two into a single workflow, where the `architecture` is passed in as a param? The project workflows can then just call this twice, with a different param each time?

File could then be called `dotnet_build-and-push-docker-image`?

Reply:
Yep

Reply:
Done